src/test/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessorTest.java

```diff
@@ -54,4 +52,10 @@ void shouldReturnEmptyWhenNoFilesToReview() {
         assertThat(reviewResult.getViolations()).isEmpty();
     }
 
+    @Test
+    void shouldLoadSpotbugsPlugins() {
```

Comment:
If you want to test it you should provide some plugin. Then you should check that this plugin was loaded. I would expect three tests here with two different configurations:

1. empty `spotbugs.plugins.location`
2. `spotbugs.plugins.location` is present but directory is empty (just place some `.gitkeep` file there)
3. `spotbugs.plugins.location` is present and plugin is loaded - you should check in some plugin for test resources